src/pages/Homepage.jsx

```diff
@@ -0,0 +1,12 @@
+import { Categories, Hero } from "components";
+import React from "react";
+import { Fragment } from "react/cjs/react.production.min";
```

Comment:
you can directly use fragments  i.e <></> ,no need to import.

Reply:
yes, made that change